config/filter.d/exim-common.conf

```diff
@@ -9,7 +9,7 @@ after = exim-common.local
 
 [Definition]
 
-host_info = H=([\w.-]+ )?(\(\S+\) )?\[<HOST>\](:\d+)? (I=\[\S+\]:\d+ )?(U=\S+ )?(P=e?smtp )?
+host_info = (H=([\w.-]+ )?(\(\S+\) )?)?\[<HOST>\](:\d+)? (I=\[\S+\](:\d+)? )?(U=\S+ )?(P=e?smtp )?
```

Comment:
just to please ya @sebres converted those I have spotted into non-capturing ones (so bye bye a somewhat minimalistic patch to resolve an issue)